package.json

```diff
@@ -37,7 +37,10 @@
     "test": "CI=true lerna run test --parallel"
   },
   "devDependencies": {
+    "@polkadot/api-derive": "^0.99.1",
     "@polkadot/dev": "^0.31.1",
+    "@polkadot/extension-dapp": "^0.14.1",
+    "@polkadot/types": "^0.99.1",
```

Comment:
I'm not sure why these are added

Reply:
techincally `extension-dapp` is  a peerDependency but it wouldn't compile as that for some reason, and `@polkadot/types` was needed for, well, the types (it's not included anywhere else).